files/zh-cn/web/api/document/startviewtransition/index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Document: startViewTransition() method"
+short-title: startViewTransition()
+slug: Web/API/Document/startViewTransition
+page-type: web-api-instance-method
+status:
+  - experimental
+browser-compat: api.Document.startViewTransition
+---
+
+{{APIRef("Document")}}{{SeeCompatTable}}
+
+{{domxref("View Transitions API", "View Transitions API", "", "nocode")}} 的 **`startViewTransition()`** 方法开始一个新的视图过渡，并返回一个 {{domxref("ViewTransition")}} 对象来表示它。
+
+当调用 `startViewTransition()` 时，将按照 [视图过渡过程](/zh-CN/docs/Web/API/View_Transitions_API#视图过渡过程) 中所述的步骤序列进行。
```

Comment:
Please remove the spaces, you could check the [rules listed here](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#%E4%B8%AD%E6%96%87%E5%92%8C%E6%8B%89%E4%B8%81%E8%AF%AD%E7%B3%BB%E6%96%87%E5%AD%97%E9%97%B4%E5%8A%A0%E7%A9%BA%E6%A0%BC)

```suggestion
当调用 `startViewTransition()` 时，将按照[视图过渡过程](/zh-CN/docs/Web/API/View_Transitions_API#视图过渡过程)中所述的步骤序列进行。
```